lib/desired-caps.js

```diff
@@ -165,6 +165,9 @@ let desiredCapConstraints = _.defaults({
   screenshotQuality: {
     isNumber: true
   },
+  skipLogCapture: {
```

Comment:
This is not necessary if the desired capability is specified in `appium-ios-driver`.

Reply:
Will remove here after https://github.com/appium/appium-ios-driver/pull/355 and publish new version.
(In this PR or in another PR)